### ISSUE TITLE
Allow cutting a closed line without converting it into a multipolygon

### ISSUE THREAD

**Issue by yaugenka** [openstreetmap/iD#4866]
Currently when you cut a closed line like natural=water it gets converted into a multipolygon.
It would be nice to have a possibility to prevent the auto-conversion by holding say alt key.
This operation is needed when you want to merge adjecent areas or redraw part of the line.

**Comment by bhousel** [comment]
If we had a dedicated operation like "unclose" which would break the closed line at the selected point - would that be ok? 

I'm leaning more towards that because:
- we already overuse the <kbd>alt</kbd> key
- there is existing code that tries very hard to keep closed ways closed that would need to change

**Comment by yaugenka** [comment]
It if will be possible to unclose at an attached node (see example screenshot below) then I think the operation wound differ much from cutting and hence should be ok.
![openstreetmap](https://user-images.githubusercontent.com/14937001/37242877-5fa1c0f4-2481-11e8-9739-9ef371cfc8f8.png)

**Comment by bhousel** [comment]
Ok.. I get that it can sometimes be useful to break a polygon and put it back together, but I'm ok with this being hard to do in iD.  Maybe we can consider this as an enhancement following #2929

**Comment by slhh** [comment]
@yaugenka 
You don't seem to need the requested operation, but an easy way to modify areas.
I think an area merge opearation (#3074, #435), an operation to redraw area outlines (as discussed in #2272) and a ´n area split along lines (as discussed in #969) would cover your requirements.

@bhousel 

> If we had a dedicated operation like "unclose" which would break the closed line at the selected point - would that be ok?
I haven't thought about this operation in detail yet, but something like that makes sense, even if not required for the mentioned usecases.
The remaining usecases would be rare, and it is structure breaking operation, we should avoid to encourage user to abuse it too much.

Therefore, I don't like to clutter up the normal context menu with such an operation, but suggest the following:

We do currently use a right-click to open the context menu and a left click to execute the operation, but a right click on the operation icon does nothing.
Instead, we can let the right click open a second level context menu containing modified operations where applicable. Availability of such operation might be checked when opening the second menu level to avoid degrading the performance when using the normal operations only. This would also allow operations with quite expensive availability checks.

**Comment by yaugenka** [comment]
Another use case to mention is fixing landuse areas which are drawn attached to a highway. Instead of detaching and dragging off each node, it would be much easier to just cut and delete the attached segment and then draw a new one along the highway.

In general, to my mind, the cut operation is one of those core things which advanced users might prefer over the bulk operations like the above referenced merge feature. So I agree with @bhousel that putting it into the list of normarly disabled operations is a good idea.

**Comment by slhh** [comment]
> Another use case to mention is fixing landuse areas which are drawn attached to a highway. Instead of detaching and dragging off each node, it would be much easier to just cut and delete the attached segment and then draw a new one next to the highway.

This is an important usecase, but it can be nicely covered by #2272. Why should users split and remergeif they can just redraw the area or multipolygon outline along the highway, especilly if the draw mode support following the highway in parallel https://github.com/openstreetmap/iD/issues/1614#issuecomment-372509450 . Imagine how complicated your approach becomes when the landuse is a multipolygon build up from highway segments. Such complicated things should be handeled by the software, but not by the user. With a flexible implementation of #2272, the user wouldn't even have to care whether it is a multipolygon or not.

> In general, to my mind, the cut operation is one of those core things which advanced users might prefer over the bulk operations like the above referenced merge feature. So putting it into the list of normarly disabled operations as suggested by @bhousel is a good idea.

The real remaining usecase would be converting an area feature into a line feature, but we might prefer a preset solution for this task.

For any other remaining usecases the multipolygon conversion doesn't seem to be a real problem in my opinion, at least not for advanced users. To work with this well, some operations are currently missing.
1. A copy tags operation
2. An operation to convert a multipolygon back into a closed way area.
We could make the merge operation available for a single selected multipolygon, and let it convert any splitted rings into closed way ones and if applicable (one ring only) convert the result into a closed way area.

**Comment by yaugenka** [comment]
If I understand the idea of #2272 feature correctly, it will let you re-enter area draw mode to add new nodes to it. In our case the user would still end-up with a necessity to cut the incorrect segment after drawing (with or without the follow a path ability) the correct one.
![continue](https://user-images.githubusercontent.com/14937001/37338861-bd3fc428-26c9-11e8-9ee5-914adae18aaa.gif)

The cut feature in combination with #4245 would even exclude the necessity to redraw. You would cut, disconnect, move and merge.

**Comment by boothym** [comment]
>Another use case to mention is fixing landuse areas which  are drawn attached to a highway. Instead of detaching and dragging off  each node, it would be much easier to just cut and delete the attached  segment and then draw a new one next to the highway.

That could also be achieved by allowing you to delete/disconnect landuse nodes while hiding roads, similar to #4622
